README.md

```diff
@@ -1,79 +1,189 @@
+If this offends you you are free to use another Mastodon client.
 
-**A:** Because the official app includes some non-free components, and is missing some key features like bookmarks.
-##
+## Timelines
 
-**Q:** How do I send a direct message?
+### Why is the count of likes and boosts different to the web?
 
-**A:** Create a toot, mention the user and set the visibility (World icon) to Direct.
-##
+This is because of how Mastodon works. Likes and boosts do not federate perfectly; if you and the other user are on different servers you will see different numbers between Tusky and the website.
 
-Template:
-**Q:**
+### How do I see the Federated timeline?
 
-**A:**
-##
+From your home timeline:
+
+- Open "Account Preferences" (swipe from the left edge, or tap your profile picture to see the option)
+- Tap "Tabs"
+- Tap the "+" button (bottom right)
+- Choose "Federated" from the menu
+
+You can adjust the order of tabs on this screen too.
+
+### How do I hide boosted posts from my timeline?
+
+- Open "Preferences" (swipe from the left edge, or tap your profile picture to see the option)
+- Tap the "Filters > Tabs" section
+- Unselect the "Boosts" option
+
+### How do I hide replies to posts from my timeline?
+
+- Open "Preferences" (swipe from the left edge, or tap your profile picture to see the option)
+- Tap the "Filters > Tabs" section
+- Unselect the "Replies" option
+
+### How do I follow a hashtag?
+
+- Tap the hashtag in a post to view all posts with this hashtag
+- Tap the "follow" icon at the top right
+
+### How do I mute a hashtag?
+
+> This feature is available in Tusky 21 or above (including Nightly). To check the Tusky version you are using open the "About" menu.
+
+- Tap the hashtag in a post to view all posts with this hashtag
+- Tap the "mute" icon at the top right
+
+## Posting
+
+### How do I edit a post?
+
+> This feature is available in Tusky 21 or above (including Nightly). To check the Tusky version you are using open the "About" menu.
+
+Find the post you want to edit, then:
+
+- Tap the "..." menu at the bottom-right of the post
+- Choose "Edit"
+- Tap "Toot" to send the edited post (replacing the original)
+
+### How do I send a direct message?
+
+> Important: Direct messages on Mastodon are not encrypted, and can be read by the admins of your instance. Be careful sharing any private information on Mastodon.
+
+- Create a new post
+- @-mention the user you want to send it to
+- Tap the "visibilty" icon (by default this looks like a globe)
+- Set the post's visibility to "Direct"
+
+## Notifications
+
+### Why are notifications less frequent with Tusky?
+
+By default, Tusky checks for new notifications approximately every 15 minutes.
+
+Tusky can also use [Unified Push](https://unifiedpush.org/) to show notifications in real-time. To do this:
+
+- Install "ntfy" (from [Google Play](https://play.google.com/store/apps/details?id=io.heckel.ntfy) or [F-Droid](https://f-droid.org/packages/io.heckel.ntfy)
+
+## Filters
```

Comment:
```suggestion
### Why does Tusky use UnifiedPush instead of Google services for notifications?

Google Services don't work on all devices and we want to give users the freedom to chose their push provider.

## Filters
```